metricflow/test/integration/test_cases/itest_dimensions.yaml

```
@@ -188,6 +188,83 @@ integration_test:
     WHERE listing__country_latest = 'us'
     GROUP BY user__home_state_latest
 ---
+integration_test:
+  name: metric_time_only
+  description: Query metric_time alone
+  model: SIMPLE_MODEL
+  group_bys: ["metric_time"]
+  check_query: |
+    SELECT
+      {{ render_date_trunc("ds", TimeGranularity.DAY) }} AS metric_time__day
+    FROM {{ source_schema }}.mf_time_spine
+    GROUP BY metric_time__day
```

There's a good chance Trino won't like this. You can either wrap the aliasing in a subquery or substitute positional numbers for the group by, so either this:

```
SELECT metric_time__day
FROM (
  SELECT {{ render_date_trunc.... }} AS metric_time__day
  FROM ...
) a
GROUP BY metric_time__day
```

Or this:
```suggestion
    GROUP BY 1
```